backend/src/http.rs

```diff
@@ -105,6 +105,12 @@ impl Service<Request<Body>> for RootService {
                 Assets::serve(&path[1..])
             }
 
+            // Since we do routing in the client, we need to serve the index for
+            // basically any path. For now, we just assume that paths with `.`
+            // in them refer to some file.
+            // TODO: this heuristic is probably BS.
```

Comment:
I think it's actually worth considering serving the static stuff from a folder, as in `domain.tld/static`. That way we can reliably 404 everything that's not actually an asset, and just redirect everything in `/` that is not a known path to `index.html` and let the frontend side routing take care of 404-ing unknown routes. :thinking:

Reply:
Serving static stuff from folder: :+1: 

Regarding routes... yes we can do that as you suggest. But I think it's worth considering teaching the backend about routes. That way it can reliably reply with 404 for unknown routes and more importantly: we did think about the backend sending some API data bundled with `index.html` right? I will open an issue about that!

Reply:
But the static files folder is "blocked" by #16

Reply:
Hmm okay, later then, I guess.